### ISSUE TITLE
Bulk Workflow Manager should support AEM WF,  Synthetic WF, and Synthetic WF w/ FAM

### ISSUE THREAD

**Issue by davidjgonzalez** [Adobe-Consulting-Services/acs-aem-commons#736]

**Comment by davidjgonzalez** [comment]
POC build

[acs-aem-commons-content-2.5.1-SNAPSHOT-min.zip](https://github.com/Adobe-Consulting-Services/acs-aem-commons/files/316372/acs-aem-commons-content-2.5.1-SNAPSHOT-min.zip)

**Comment by justinedelson** [comment]
@davidjgonzalez it sounds like you are going to be using oak:Unstructured. This means that BWM would no longer work on CRX2 installations, which 2.5.x theoretically supports. I'm totally fine with this, but we should add something to the docs saying this.

**Comment by davidjgonzalez** [comment]
@justinedelson yup - agree. 

We could look at using CRX2 supported unordered nodes instead (sling:Folder?) .. though im unsure what the effect of such flat structures may be on the overall process.
The tree build is happens once, read often, and each batch nodes is updated "frequently".

**Comment by justinedelson** [comment]
I'll leave it up to you to decide, but I think the conditional logic is adding unnecessary complexity.

**Comment by davidjgonzalez** [comment]
@justinedelson TBH im more concerned w/ the potential for negative perf impacts in CRX2 over the logic (login would be simple). I've never made super flat structures in CRX2 since we werent supposed to, and its been so long i can't recall how well proven out the pros/cons are for use-cases like this. (create one, read many, update minimally)

**Comment by justinedelson** [comment]
fair enough, but if we aren't confident in the performance on CRX2, I'd still be inclined to say that BWM doesn't work on CRX2. I can't imagine that there are that many users who are upgrading to the latest commons *and* using BWM but still on CRX2.

**Comment by bstopp** [comment]
That'd be what i'd go with too; to be on CRX2 and 2.x you'd have to still be on 6.0; such a niche likelihood, why bother to support?
